### PR TITLE
Change swagger 2.0 to 3.0.0

### PR DESCRIPTION
**Pull request by Chiemezuo**
Here's a checklist of things to be completed before this PR will be ready for a merge.

- [x] `/register` path
- [x] `/login` path
- [ ] `/logout` path
- [x] `/reset-password` path
- [x] `/update-password` path
- [x] `/user/stipend/apply` path
- [x] `/user/check` path
- [ ] `/user/stipend/application-history` path
- [x] `/user/verify` path
- [ ] `/stipend/apply` path
- [ ] `/stipend/one-click-apply/:email` path
- [x] `admin/approve-stipend` path
- [x] `admin/reject-stipend` path
- [x] `admin/notify-waitlist` path
- [x] `admin/application-window` path
- [x] `admin/close-application-window` path
- [x] `waitlist/join-waitlist` path

- [ ] Auth in swagger docs.

### CODE REVIEW

**Comment by Chiemezuo** [comment]
@edustipend-root These are the changes so far.

File: routes/StipendApplicationRoute.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const router = require("express").Router();
 // router.use(isWindowOpen)
 
 router.post("/apply", isAuthenticated, createStipendApplication);
+// I think they should be authenticated for this
```
Comment (edustipend-root) [inline routes/StipendApplicationRoute.js:16]: Yes they will be. It's part of what I'll be working on for February release @Chiemezuo

**Review by edustipend-root — APPROVED** [review]

**Comment by edustipend-root** [comment]
@Chiemezuo, I've approved the PR but please check the merge conflicts

**Comment by Chiemezuo** [comment]
@edustipend-root I'll handle the merge conflicts.

**Comment by Chiemezuo** [comment]
@edustipend-root Help me check if the merge conflict has been fixed.

**Pull request merged**